converters/fromZigbee.js

```diff
@@ -4010,7 +4010,12 @@ const converters = {
             case tuya.dataPoints.moesChildLock:
                 return {child_lock: value ? 'LOCK' : 'UNLOCK'};
             case tuya.dataPoints.moesHeatingSetpoint:
-                return {current_heating_setpoint: value};
+                temperature = value & 1<<15 ? value - (1<<16) + 1 : value;
+                if (!['_TZE200_5toc8efa'].includes(meta.device.manufacturerName)) {
```

Comment:
You say currently for the `_TZE200_5toc8efa` the `current_heating_setpoint` is 10 times lower than expected. With this fix you only modify the value when manufacturer name != `manufacturerName`. How can this fix your issue?